net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="mirror://sourceforge/nfs/${P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="alpha amd64 arm arm64 hppa ia64 ~mips ppc ppc64 s390 sh sparc x86"
+KEYWORDS="~alpha amd64 arm ~arm64 hppa ~ia64 ~mips ppc ppc64 ~s390 ~sh sparc x86"
```

Comment:
Hm, well it seems a bit odd to have both 1.3.1 and 1.3.2 ebuilds in the overlay. Or if they are both there both should be in sync and have the same fixes (symlinks etc) in them. Either way the arm64 keywords can remain unstable for now.